transformers/tokenization_utils.py

```diff
@@ -352,7 +353,8 @@ def _from_pretrained(cls, pretrained_model_name_or_path, *init_inputs, **kwargs)
                 if file_path is None:
                     resolved_vocab_files[file_id] = None
                 else:
-                    resolved_vocab_files[file_id] = cached_path(file_path, cache_dir=cache_dir, force_download=force_download, proxies=proxies)
+                    resolved_vocab_files[file_id] = cached_path(file_path, cache_dir=cache_dir,
+                            force_download=force_download, proxies=proxies, xla_device=xla_device)
```

Comment:
indentation seems weird here, as well as some other places.

Reply:
Updated. Let me know if it still looks weird.